### ISSUE TITLE
*_spec.txt has v_app instead of v_rad for cube with 'CTYPE3=VRAD'

### ISSUE THREAD

**Issue by kmhess** [SoFiA-Admin/SoFiA-2#81]
I have a cube of ngc 1365 from Thijs which has ctype3=vrad, however the spectral column in the resulting *_spec.txt file is labeled as 'v_app' rather than as 'v_rad'.  Based on a previous discussion, which pointed to the code where this happens, I think this is the incorrect behavior:
https://github.com/SoFiA-Admin/SoFiA-2/issues/60#issuecomment-1033639049

However, I don't know how SoFiA finds the CTYPE...it seems like 'v_app' should only appear if ctype3 = 'velo' or 'felo'.  Therefore, I wonder if it's related to the following issue where I found that astropy.io.fits was converting vrad to velo-hel under-the-hood and without warning.
https://github.com/kmhess/SoFiA-image-pipeline/issues/49#issuecomment-1147260022

**Comment by SoFiA-Admin** [comment]
Thanks for reporting this issue. I have had a look at the source code and have absolutely no idea how you could possibly get “v_app” in the spectrum, as “v_app” is only set if `CTYPE3` is set to `VELO` (see source code below).
https://github.com/SoFiA-Admin/SoFiA-2/blob/bf4e695871febb69bbe006cc37ae8239aad5d3ee/src/DataCube.c#L5453-L5477
Likewise, if `CTYPE3 = VRAD`, then the label will be set to “v_rad” according to the source code. Are you absolutely sure that the value of `CTYPE3` in the data cube that you actually feed into SoFiA is set to `VRAD` and not `VELO`?

**Comment by kmhess** [comment]
Yeah...working with Thijs we realized that this confusion has actually arisen because CASA automatically converts the header to VRAD in all cases.  So it seems there is no bug here.  I had assumed it was showing me the true FITS header, however we have now crosschecked with astropy and ds9.  Very annoying that the tool built to work with cubes is the one that's wrong. :/
https://github.com/CARTAvis/carta-frontend/issues/1877
